src/Sarif.Multitool.FunctionalTests/Rules/StepValuesMustFormOneBasedSequenceTests.cs

```diff
@@ -31,5 +31,11 @@ public void StepValuesMustFormOneBasedSequence_InvalidStepValues()
         {
             Verify(new StepValuesMustFormOneBasedSequence(), "InvalidStepValues.sarif");
         }
+
+        [Fact(DisplayName = nameof(StepValuesMustFormOneBasedSequence_MultipleThreadFlows))]
```

Comment:
>Fact [](start = 9, length = 4)

thank you for not using a theory for this! facts are faster to process

Reply:
Yes, you've sold me on that. I had no idea it was so slow compared even to file i/o.

---
In reply to: [210452008](https://github.com/Microsoft/sarif-sdk/pull/1018#discussion_r210452008) [](ancestors = 210452008)

Reply:
I wonder whether it was the enumeration over the test case data specifically that caused the problem. might poke around on it at a future time.

---
In reply to: [210453847](https://github.com/Microsoft/sarif-sdk/pull/1018#discussion_r210453847) [](ancestors = 210453847,210452008)